content/updates/starting-now-sandboxes-expire-after-90-days.md

```diff
@@ -0,0 +1,14 @@
++++
+title: "Starting now, Sandboxes expire after 90 days"
+date: 2017-12-06
++++
+
+In our monthly update from November, we mentioned we would start expiring sandboxes after 90 days. Today, we're starting that clock. Here's what to expect.
+ 
+
+* Any sandboxes applications you have right now will be deleted 90 days from today.
+* The apps themselves will be deleted and any managed services attached to them will be disconnected.
+* You will not lose your cloud.gov account.
+* Starting five days before we delete your sandbox, we'll send you an email reminder every day unless you delete your sandbox yourself
+
+If you have an application in your sandbox that you need to keep online for longer than 90 days, you should consider purchasing a paid cloud.gov account. Any prototyping or production account will allow you to host apps for any amount of time. Contact us to learn how to get started with a paid plan.
```

Comment:
Or users can redeploy their applications after their sandbox is cleared. I think we should try to be clearer on this point: users can continue to use their sandbox after we clear its contents--they'll just have to deal with the inconvenience of losing their applications and services every 90 days.

Reply:
Do we want to advertise that, though? Seems like we'd rather push people toward buying cloud.gov if they need to keep their apps running persistently?

Reply:
I think the text that @pburkholder pulled from existing docs addresses this issue pretty well. Can we check in that text, fix the casing nit @mogul pointed out, and merge?